app/poi_api/service.py

```diff
@@ -0,0 +1,65 @@
+from flask import jsonify
+from typing import List, Dict
+from .poi_suggestion import POI_suggestion
+from .poi import POI
+from .errors import POINotFoundError, InvalidPOISuggestionError
+from app import common
+from datetime import datetime
+from app.firebase import firestore_db, POI_COLLECTION, POI_PROPOSAL_COLLECTION
+
+poi_collection = firestore_db.collection(POI_COLLECTION)
+poi_proposal_collection = firestore_db.collection(POI_PROPOSAL_COLLECTION)
+
+
+def list_POI() -> List[POI]:
+    return [POI.from_dict(doc.to_dict()) for doc in poi_collection.stream()]
+
+
+def get_details_for_POI(poi_id: str) -> POI:
+    poi_data = poi_collection.document(poi_id).get()
+    if not poi_data.exists:
+        raise POINotFoundError(poi_id)
+    return POI.from_dict(poi_data.to_dict())
+
+
+def new_POI_suggestion(poi_suggestion: Dict[str, str]) -> int:
+    poi_suggestion_ref = poi_proposal_collection.document()
+    # Generate id for poi suggestion document
+    pid = poi_suggestion_ref.id
+    poi_suggestion_instance = _create_POI_suggestion(pid, poi_suggestion)
+    _save_POI_suggestion(poi_suggestion_instance, pid)
+    return 204

```

Comment:
Why is a service method returning an integer?

Reply:
oops nice catch I forgot to change it